diff -ruN sketchybar.old/colors.lua colors.lua
--- sketchybar.old/colors.lua	2024-06-09 15:46:47
+++ colors.lua	2024-06-09 16:01:35
@@ -1,25 +1,25 @@
 return {
-  black = 0xff181819,
-  white = 0xffe2e2e3,
-  red = 0xfffc5d7c,
-  green = 0xff9ed072,
-  blue = 0xff76cce0,
-  yellow = 0xffe7c664,
-  orange = 0xfff39660,
-  magenta = 0xffb39df3,
-  grey = 0xff7f8490,
+  black = 0xff181926,
+  white = 0xffcad3f5,
+  red = 0xffed8796,
+  green = 0xffa6da95,
+  blue = 0xff8aadf4,
+  yellow = 0xffeed49f,
+  orange = 0xfff5a97f,
+  magenta = 0xffc6a0f6,
+  grey = 0xff939ab7,
   transparent = 0x00000000,
 
   bar = {
-    bg = 0xf02c2e34,
-    border = 0xff2c2e34,
+    bg = 0xff1e1e2e,
+    border = 0xff494d64,
   },
   popup = {
-    bg = 0xc02c2e34,
-    border = 0xff7f8490
+    bg = 0xff1e1e2e,
+    border = 0xffcad3f5
   },
-  bg1 = 0xff363944,
-  bg2 = 0xff414550,
+  bg1 = 0x603c3e4f,
+  bg2 = 0x60494d64,
 
   with_alpha = function(color, alpha)
     if alpha > 1.0 or alpha < 0.0 then return color end
diff -ruN sketchybar.old/items/calendar.lua items/calendar.lua
--- sketchybar.old/items/calendar.lua	2024-06-09 15:46:47
+++ items/calendar.lua	2024-06-09 16:02:27
@@ -44,5 +44,5 @@
 sbar.add("item", { position = "right", width = settings.group_paddings })
 
 cal:subscribe({ "forced", "routine", "system_woke" }, function(env)
-  cal:set({ icon = os.date("%a. %d %b."), label = os.date("%H:%M") })
+  cal:set({ icon = os.date("%a. %d %b."), label = os.date("%I:%M") })
 end)
